TestCPP11.cmake

```diff
@@ -98,6 +100,13 @@ while(TESTS_CPP11)
 
   if(CXX_${TEST_CPP11_NAME}_SUPPORTED)
     add_definitions(-DCXX_${TEST_CPP11_NAME}_SUPPORTED)
+    list(APPEND __supported ${TEST_CPP11_name})
+  else()
+    list(APPEND __unsupported ${TEST_CPP11_name})
   endif()
 endwhile()
 
+if(__unsupported)
+  set(__unsupported " UNSUPPORTED: ${__unsupported}")
+endif()
+message(STATUS "Tested C++11 features: ${__supported}${__unsupported}")
```

Comment:
I remember removing this because of spam. It would be nice if there would be message(VERBOSE) which you can enable if you want it. You really think it is useful/needed?